frontend/src/APIClients/queries/StoryQueries.ts

```diff
@@ -114,22 +114,38 @@ export const buildHomePageStoriesQuery = (
   return result as QueryInformation;
 };
 
+const COMMENT_FIELDS = `
+    id
+    commentIndex
+    content
+    storyTranslationContentId
+    `;
+
 export const buildCommentsQuery = (
   storyTranslationId: number,
+  resolved: boolean | null,
 ): QueryInformation => {
-  // TODO: add more query parameters
+  let queryString = gql`
```

Comment:
Does it dislike `if`/`else`?

Reply:
it starts to look quite hairy with ternary operators/conditions within the gql string 
```
let queryString = gql`
    query ComentsByStoryTranslation {
      commentsByStoryTranslation( storyTranslationId: ${storyTranslationId}${
    resolved === true || (resolved === false && ", resolved:" + resolved)
  } ){
        ${COMMENT_FIELDS}
      }
    }
  `;
```
but typescript lint prefers string templates to concatenation (regarding `", resolved:" + resolved`)